schemas/bedrock-authn-token-http.js

```diff
@@ -0,0 +1,171 @@
+/*!
+ * Copyright (c) 2022 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const postToken = {
+  title: 'postToken',
+  type: 'object',
+  additionalProperties: false,
+  anyOf: [
+    {required: ['email']},
+    {required: ['account']}
+  ],
+  properties: {
+    email: {
+      type: 'string'
+    },
+    account: {
+      type: 'string'
+    },
+    hash: {
+      type: 'string'
+    },
+    authenticationMethod: {
+      type: 'string'
+    },
+    requiredAuthenticationMethods: {
+      type: 'array'
+    },
+    typeOptions: {
+      type: 'object',
+      properties: {
+        entryStyle: {
+          type: 'string'
+        }
+      }
+    }
+  }
+};
+
+const getTokensQuery = {
+  title: 'getTokensQuery',
+  type: 'object',
+  additionalProperties: false,
+  required: ['email'],
+  properties: {
+    type: {
+      type: 'string',
+      enum: ['password', 'nonce']
+    },
+    email: {
+      type: 'string'
+    }
+  }
+};
+
+const postAuthenticate = {
+  title: 'postAuthenticate',
+  type: 'object',
+  additionalProperties: false,
+  required: ['type'],
+  anyOf: [
+    {required: ['email']},
+    {required: ['account']}
```

Comment:
Aren't there some other combinations there? Like, if you use `type = totp` you also have to use `challenge` or `hash` or something like that? This doesn't seem strict enough.

Reply:
I added in where some combination of both `account` or `email` and `hash` or `challenge` must be present.